src/wymeditor/iframe/default/wymiframe.html

```diff
@@ -21,5 +21,5 @@
 <title>WYMeditor iframe</title>
 <link rel="stylesheet" type="text/css" media="screen" href="wymiframe.css" />
 </head>
-<body class="wym_iframe"></body>
+<body></body>
```

Comment:
That change means that the `wym_iframe` class will be added to the Iframe upon instantiation.

This is why I've removed it from the Iframe HTML—it is just redundant and means more code and one less thing you must do to have a working custom Iframe.

I don't see backward incompatibility.